minvws/nl-kat-coordination
# Add documentation for S3 Support

### Changes

Added a new document that describes 

### Issue link

No issue is created. This is an addition for #3191. S3 support has been implemented but there was no documentation yet on how to enable this feature.

### Demo

![image](https://github.com/user-attachments/assets/6c551bdb-59a3-4d14-a587-6e1f50939e02)


### QA notes

To see the docs I ran `make docs` and then ran `python -m http.server 8021` inside `docs/_build`. 

---

### Additional notes

I have noticed whenever I create a new documentation, this documentation sometimes does not show in my sidebar. It is seemingly random, but it might have to do with my local configuration.

Picture:
![image](https://github.com/user-attachments/assets/71d91340-0423-4578-a7b6-9a90523ffcb6)
